pkg/Library/Mediapipe/MediapipeService.js

```diff
@@ -42,9 +42,10 @@ export const MediapipeService = {
   async renderSticker({data, sticker, target}) {
     const faceLandmarks = data?.faceLandmarks;
     const realTarget = Resources.get(target);
+    console.log(sticker);
     if (faceLandmarks && realTarget) {
       const ctx = realTarget.getContext('2d');
-      Mediapipe.renderSticker(ctx, {faceLandmarks});
+      Mediapipe.renderSticker(ctx, {faceLandmarks}, sticker ? await loadImage(sticker.url) : masque);
```

Comment:
This is not right, 'sticker' should handled exactly the same as 'target' .

Reply:
Done!